File: Classes/EVEOnlineAPI/Corporation/EVECorpStandings.m
//
//  EVECorpStandings.m
//  EVEOnlineAPI
//
//  Created by Артем Шиманский on 11.08.15.
//
//

#import "EVECorpStandings.h"

@implementation EVECorpStandings

+ (NSDictionary*) scheme {
	static NSDictionary* scheme = nil;
	if (!scheme)
		scheme = @{@"corporationNPCStandings":@{@"type":@(EVEXMLSchemePropertyTypeObject), @"class":[EVEStandingsNPCStandings class]}};
	
	return scheme;
}

@end

File: EVEAPI/EVEAPI/EVECorpStandings.swift
//
//  EVECorpStandings.swift
//  EVEAPI
//
//  Created by Artem Shimanski on 30.11.16.
//  Copyright © 2016 Artem Shimanski. All rights reserved.
//

import UIKit

public class EVECorpStandings: EVEResult {
	
	public required init?(dictionary:[String:Any]) {
		super.init(dictionary: dictionary)
	}
	
	public required init?(coder aDecoder: NSCoder) {
		super.init(coder: aDecoder)
	}
	
	override public func scheme() -> [String:EVESchemeElementType] {
		return [
			"npcStandings":EVESchemeElementType.Object(elementName: "corporationNPCStandings", type: EVEStandingsNPC.self, transformer: nil),
		]
	}
}
